socket_can/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
The CMakeLists.txt in socket_can directory can be removed as that folder was removed here: #49

Reply:
Removed and rebased.